script/Arcueid/Day 7/07_00_ARC07_3D.txt

```diff
@@ -573,8 +577,9 @@ In retrospect, maybe I was a bit too cruel...
 [sha:17727160c5148ae0c1ff22a9333aeb8218c1cadd]{
 -- Page 271, Offset 17233.
 -- けど、あいつがあんなに怒ったんだから俺が悪い気もする。……それも全面的に。なんか最低っぽく。
+// italics candidate, it is
 Though, seeing how upset she is, I feel like I'm the one at fault...
- Actually, it is completely my fault.
+ Actually, it's completely my fault.
```

Comment:
If you're going to italicise, then keep it in full and don't use a contraction, because the italics should be for 'is'.

Reply:
Yep, this is already the plan. 👍